docs/source/docs/examples/aimingatatarget.md

```diff
@@ -1,16 +1,16 @@
 # Aiming at a Target
 
-The following example is from the PhotonLib example repository ([Java](https://github.com/PhotonVision/photonvision/tree/master/photonlib-java-examples/aimattarget)/[C++](https://github.com/PhotonVision/photonvision/tree/master/photonlib-cpp-examples/aimattarget)).
+The following example is from the PhotonLib example repository ([Java](https://github.com/PhotonVision/photonvision/tree/master/photonlib-java-examples/aimattarget)).
 
 ## Knowledge and Equipment Needed
 
 - Robot with a vision system running PhotonVision
-- Target
-- Ability to track a target by properly tuning a pipeline
+- AprilTag 7
+- Ability to track a AprilTags with a well-tuned pipeline
 
 ## Code
 
-Now that you have properly set up your vision system and have tuned a pipeline, you can now aim your robot/turret at the target using the data from PhotonVision. This data is reported over NetworkTables and includes: latency, whether there is a target detected or not, pitch, yaw, area, skew, and target pose relative to the robot. This data will be used/manipulated by our vendor dependency, PhotonLib. The documentation for the Network Tables API can be found {ref}`here <docs/additional-resources/nt-api:Getting Target Information>` and the documentation for PhotonLib {ref}`here <docs/programming/photonlib/adding-vendordep:What is PhotonLib?>`.
+Now that you have properly set up your vision system and have tuned a pipeline, you can now aim your robot/turret at an AprilTag using the data from PhotonVision. This data is reported over NetworkTables and includes: latency, whether there is a AprilTag detected or not, pitch, yaw, area, skew, and target pose relative to the robot. This data will be used/manipulated by the vendor dependency, PhotonLib. The documentation for the Network Tables API can be found {ref}`here <docs/additional-resources/nt-api:Getting Target Information>` and the documentation for PhotonLib {ref}`here <docs/programming/photonlib/adding-vendordep:What is PhotonLib?>`.
 
 For this simple example, only yaw is needed.
 
```

Comment:
I cant comment on line 17 here for some reason but goal should be reworded to AprilTag? again, pedantic not super important

Reply:
Tweaked for consistency. Presumably they'll put an apriltag above a goal, but the GDC gods are fickle